qiskit_dynamics/pulse/pulse_to_signals.py

```diff
@@ -44,13 +44,15 @@ class InstructionToSignals:
     :class:`ScheduleBlock` must first be converted to :class:`Schedule` using the
```

Comment:
a few lines up, wrap Signals in a :class:, or lower-case it

Reply:
lower cased it.